hadoop-hdds/docs/content/interface/ReconApi.md

```diff
@@ -232,7 +232,174 @@ Returns the UnhealthyContainerMetadata objects for all the unhealthycontainers.
 Returns the UnhealthyContainerMetadata objects for the containers in the givenstate.
 Possible unhealthy container states are `MISSING`, `MIS_REPLICATED`,`UNDER_REPLICATED`, `OVER_REPLICATED`.
 The response structure is same as `/containers/unhealthy`.
-     
+
+## Namespace Metadata
+
+### GET /api/v1/namespace/summary
+
+**Parameters**
+
+* path
+
+   The path request in string without any protocol prefix.
+
+**Returns**
+
+Returns a basic summary of the path, including entity type and aggregate count 
+of objects under the path.
+
+`status` is `OK` if path exists, `PATH_NOT_FOUND` otherwise.
+
+Example: /api/v1/namespace/summary?path=/
```

Comment:
Please include another example for a non root path.

Reply:
Thanks @kerneltime for taking a look! Do we need examples of non-root path for all endpoints?